modules/db.rb

```diff
@@ -0,0 +1,21 @@
+module DB
+  YAML_FILE_PATH = 'library_db.yaml'.freeze
+
+  class << self
+    def store(entity)
+      File.open(YAML_FILE_PATH, 'a') { |file| file.write entity.to_yaml }
+    end
+
+    def load(entities)
+      records = YAML.load_stream(File.read(YAML_FILE_PATH))
```

Comment:
`load` - should return a similar hash(like entities) with loaded data.
It's not ok to add them to instance variables.